src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1260,18 +1532,38 @@ private Task<INetworkInterface> AssociateNicWithNetworkSecurityGroupAsync(INetwo
                 () => networkInterface.Update().WithExistingNetworkSecurityGroup(networkSecurityGroup).ApplyAsync()
             );
 
+        private string GetInitSqlString()
+        {
+            return Utility.PersonalizeContent(new[]
+            {
+                new Utility.ConfigReplaceRegExItemText("^CREATE USER 'cromwell'@'%'.*$", String.Empty, RegexOptions.Multiline),
+            }, "scripts", "mysql", "init-user.sql");
+        }
+
         private Task ExecuteQueriesOnAzureMySqlDb(ConnectionInfo sshConnectionInfo, Server mySQLServer)
         => Execute(
             $"Executing scripts on cromwell_db.",
             async () => {
                 await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"sudo apt install -y mysql-client");
-                var initScript = Utility.PersonalizeContent(new[]
-                {
-                    new Utility.ConfigReplaceRegExItemText("^CREATE USER 'cromwell'@'%'.*$", String.Empty, RegexOptions.Multiline),
-                }, "scripts", "mysql", "init-user.sql");
+                var initScript = GetInitSqlString();
                 return await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"mysql -u cromwell -p'{configuration.MySqlServerPassword}' -h {mySQLServer.FullyQualifiedDomainName} -P 3306 -D cromwell_db -e \"{initScript}\"");
             });
 
+        private Task ExecuteQueriesOnAzureMySqlDbFromK8(Server mySQLServer)
+            => Execute(
+                $"Executing scripts on cromwell_db.",
+                async () => 
+                {
+                    var initScript = GetInitSqlString();
+
+                    var commands = new List<string[]>() 
+                    {
+                        new string[] { "sudo apt install -y mysql-client" } ,
+                        new string[] { $"mysql -u cromwell -p'{configuration.MySqlServerPassword}' -h {mySQLServer.FullyQualifiedDomainName} -P 3306 -D cromwell_db -e \"{initScript}\"" }
```

Comment:
I think you mentioned on the call you couldn't quite get the init script to work, right? If that's the case we should file a follow up issue and leave a commment here in the format `// (Bug #<>: Description)`